updated /tutorials/security/seccomp.md (#35902)
kubernetes/website: content/en/docs/tutorials/security/seccomp.md

* Observe behavior when a missing profile is specified
* Observe a violation of a seccomp profile
* Learn how to create fine-grained seccomp profiles

> * Learn how to apply the default seccomp profile to a container

Reviewer: The original text is correct, I think.
It refers to this section (https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads). Those profiles are runtime default seccomp profiles.

> * Learn how to apply the default seccomp profile for container runtime

Reviewer 2: Maybe?

```suggestion
* Learn how to apply the default seccomp profile to a container
```

kubectl delete pod fine-pod --wait --now
```

> ## Create Pod that uses the default seccomp profile for container runtime

Reviewer 2: I think the change makes it harder to understand. The "default profile" belongs to the container runtime.
The Seccomp Profile is not for the container runtime, but is rather used by it to restrain the container process.

```suggestion
## Create Pod that uses the container runtime default seccomp profile
```
